src/App.tsx

```diff
@@ -1,61 +1,111 @@
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
 import './App.scss';
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+const getUserById = (id: number): User | null => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+const todosWithUsers: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId) || null,
```

Comment:
```suggestion
  user: getUserById(todo.userId),
```
`getUserById` returns null if user not found, so it's redundant